csharp/test/Ice/hold/AllTests.cs

```diff
@@ -135,11 +135,14 @@ public static async Task allTests(global::Test.TestHelper helper)
                     var outOfOrderTcs = new TaskCompletionSource();
                     TaskCompletionSource sentTcs = null;
                     Task result = null;
+
+                    // We use the same proxy for all oneway calls.
+                    holdSerializedOneway = (Test.HoldPrx)holdSerialized.ice_oneway();
```

Comment:
That's the only test affected by this proxy refactoring. We did the same in C++.